src/helpers/index.tsx

```diff
@@ -93,30 +94,37 @@ function getSohmTokenImage() {
   return <SvgIcon component={SOhmImg} viewBox="0 0 100 100" style={{ height: "1rem", width: "1rem" }} />;
 }
 
-export function getOhmTokenImage(w, h) {
-  h !== null ? (h = `${h}px`) : "32px";
-  w !== null ? (w = `${w}px`) : "32px";
-  return <SvgIcon component={OhmImg} viewBox="0 0 32 32" style={{ height: h, width: w }} />;
+export function getOhmTokenImage(w?: number, h?: number) {
```

Comment:
Can you just set the default parameter rathehr than checking for null?
`getOhmTokenImage(w=32, h=32)`